guides/rest_doc.md

```diff
@@ -21,6 +21,10 @@
 ## App
 * [kintoneUtility.rest.getFormFields](#getFormFields)
 * [kintoneUtility.rest.getFormLayout](#getFormLayout)
+* [kintoneUtility.rest.postDeploy](#postDeploy)
```

Comment:
I think the function name should be more clear
+ postDeploy => postDeployAppSettings